.github/workflows/build.yaml

```diff
@@ -12,8 +12,14 @@ on:
 env:
   PUB_ENVIRONMENT: bot.github
 
+permissions:
+  contents: read
+
 jobs:
   build:
+    permissions:
+      checks: write  # for coverallsapp/github-action to create new checks
+      contents: read  # for actions/checkout to fetch code
```

Comment:
If we are setting default `contents: read` above, do we need this line here?

Reply:
That wasn't required. I removed it. Thanks